core/src/main/java/com/scalar/db/transaction/rpc/GrpcTransactionManager.java

```diff
@@ -37,7 +37,7 @@
 import org.slf4j.LoggerFactory;
 
 @ThreadSafe
-public class GrpcTransactionManager extends AbstractDistributedTransactionManager {
+public class GrpcTransactionManager extends ActiveTransactionManagedTransactionManager {
```

Comment:
After this change, it looks no class will use AbstractDistributedTransactionManager directly. CMIIW, maybe just renaming AbstractDistributedTransactionManager to ActiveTransactionManagedTransactionManager is enough?